packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -179,13 +184,15 @@ export default class TurnDiscovery {
         version: ROAP.ROAP_VERSION,
         seq: meeting.roapSeq
       },
+      // @ts-ignore - fix type
       locusSelfUrl: meeting.selfUrl,
+      // @ts-ignore - fix type
       mediaId: meeting.mediaId,
       correlationId: meeting.correlationId,
       audioMuted: meeting.isAudioMuted(),
       videoMuted: meeting.isVideoMuted(),
       meetingId: meeting.id
-    });
+    } as any);
```

Comment:
Do we need this ?

Reply:
Here we are not passing `preferTranscoding` property along with other properties of the object accepted, which is mentioned in the type object of param, if that is not required we can make that property optional there. So, for now typecasting here.

Reply:
Looking at the implementation of sendRoap(), the preferTranscoding parameter should be optional, so instead of doing a cast to `any` here we should define the type for it as: `preferTranscoding?: boolean` instead of `preferTranscoding: any` that we currently have.
Same applies to all other calls to sendRoap(): I can see at least one more here in this file. I'm not sure why the other calls to sendRoap() from roap/index.ts didn't need the cast - they're also missing preferTranscoding field

Reply:
This sort of fixes we can do gradually. We can get the actual type if start investigating all the files but that is not part of this PR.

> I'm not sure why the other calls to sendRoap() from roap/index.ts didn't need the cast - they're also missing preferTranscoding field
They might not be using this property there since it is optional

Reply:
but we've already spent time looking into this and discussing it in the comments here and it's clear what the solution is: preferTranscoding is optional, so please fix it in this PR, so that someone else doesn't have to spend time on the same thing again later.

Reply:
Done